docs/search/copilot.md

```diff
@@ -0,0 +1,155 @@
+title: Sumo Logic Copilot - Feature Preview
+sidebar_label: Copilot 🤖
+keywords:
+  - copilot
+  - artificial intelligence
+  - ai
+  - machine learning
+  - ml
+---
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+import Iframe from 'react-iframe';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::info
+This is a Preview release. To learn more, contact your Sumo Logic account executive.
+:::
+
+Sumo Logic Copilot is an AI-based assistant that streamlines log analysis insights by allowing you to ask questions in plain English and provides search suggestions, all without your having to write a log query.
+
+In this document, you'll learn the recommended Copilot workflow as well as best practices.
+
+<details>
+<summary>Click here to watch a step-by-step video investigation using Copilot.</summary>
+
+<!-- add micro lesson video when published-->
+
+<Iframe url="https://player.vimeo.com/video/939372059?badge=0&amp;autopause=0&amp;player_id=0&amp;app_id=58479"
+        width="854px"
+        height="480px"
+        title="Copilot Demo"
+        id="myId"
+        className="video-container"
+        display="initial"
+        position="relative"
+        allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+        />
+
+Scenario: We receive a Sumo Logic alert about AWS CloudTrail access keys being leaked outside our organization and use Copilot to investigate.
+
+1. First, we select the data source for AWS CloudTrail audit logs.
+1. Launch our log investigation by clicking the AI-suggested insight `Count logs by eventname`, which translates the insight to a log query and renders results.
+1. Refine query by applying the suggestion `Count logs by eventname, access key and sourceIp`.
+1. Filter for Create and Authorize events, as attackers often create new resources during a breach. We find an anomaly with `AuthoritySecurityGroupIngress` events.
+1. Provide feedback to enhance the AI's accuracy stating that the queries have been accurate so far.  
+1. Add the security `groupid` to the prompt to tabulate events containing that data.
+1. Focus on a specific access key, `ABCDEFGOYCM3PIKNOVRA`, noting `PutRolePolicy` events indicating permission elevation. Multiple AWS accounts are impacted.
+1. We conclude that the access key was used to modify permissions, providing the attacker with a potential network entry point. Remediation would involve disabling the key, blocking the source IP, and further log analysis.
+</details>
+
+## Step 1: Open Copilot
+
+To start using Copilot, navigate to the **Copilot** tab on the Sumo Logic home page.
+
+<img src={useBaseUrl('img/search/copilot/copilot-tab.png')} alt="Copilot tab" style={{border: '1px solid gray'}} width="250" />
+
+## Step 2: Select a source category
+
+Click **Select Source Category** and select the data source of the log messages you want to investigate. This field is also called the source expression box.
```

Comment:
Shouldn't it be enter/select as user might want to enter his source category instead of selecting ?

Reply:
Fixed - let me know what you think